### DISCUSSION TITLE
Does neovis.js support passing in json returned by neo4j?

### DISCUSSION THREAD

**Discussion by chenkuncloud** [neo4j-contrib/neovis.js#376] [Q&A] [answered]
Hi,

Does `neovis.js` support passing in json returned by neo4j like [neo4jd3-neo4jData](https://github.com/eisman/neo4jd3#options)?

Can I directly pass in the json after neo4j query without passing in the connection information of neo4j?

**Comment by thebestnom** [comment, marked as answer]
Yep, it's a new api, you'll have to pass a function that's gets the data, check the example to see how to work with this
https://github.com/neo4j-contrib/neovis.js/blob/master/examples/simple-dataFunction-example.html

**Reply by chenkuncloud** [reply]
Thanks
